### PR TITLE
feat: KPI 177 investment by year report

### PR DESCRIPTION
**Pull request by kellycastrof**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-177)

## Types of changes

- [ ] Hotfix (Bug detected)
- [x] Feature (Issue from an Active Sprint)

## Checklist:

- [ ] Unit Test.
- [ ] Coverage.

### CODE REVIEW

File: src/views/ComparisonView/ComparisonView.jsx [hunk 1]
```diff
@@ -103,19 +102,16 @@ export function ComparisonView ({ companyComparison, peersComparison, isLoading,
...
-      topActions={
-        <ExportOption
-          buttonClass={classes.exportButton}
-          saveComparisonReport={saveComparisonReport}
-          isLoading={isLoading}
-          downloading={downloading}
-        />
-      }
-      >
+    <div style={{ overflow: 'none' }}>
```
Comment (kellycastrof) [inline src/views/ComparisonView/ComparisonView.jsx:105]: Can you add a condition here to display the comparison vs peers export data option?
This only works for comparison vs peers with the calendar year, not for investment year

File: src/views/Reports/ByYearReport.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+              onChange={(event) => onYearChange(event.target.value, 'calendar')}
+            />
+            <InvestmentYearSelector
+              nameOfSelect="Investment Year"
+              year={investYear}
+              onChange={(event) => onYearChange(event.target.value, 'investment')}
+            />
+        </Box>
+        <Box>
+            <ComparisonView
```
Comment (kellycastrof) [inline src/views/Reports/ByYearReport.jsx:63]: You can pass a variable here to know if it comes from calendar or investment year, that boolean should changes with onYearChange

File: src/views/ComparisonView/ComparisonView.jsx (hunk 1 above)
Comment (FreddyJR1995) [reply src/views/ComparisonView/ComparisonView.jsx:105]: I have added the prop to the comparison view to evaluate when export data option should be displayed.

File: src/views/Reports/ByYearReport.jsx (hunk 2 above)
Comment (FreddyJR1995) [reply src/views/Reports/ByYearReport.jsx:63]: I used the **type** variable that is already declared.

**Review by dsgarcia8 — APPROVED** [review]

**Pull request merged**